### DISCUSSION TITLE
How to perform transactions in case of postgres?

### DISCUSSION THREAD

**Discussion by shivshankardayal** [matt-42/lithium#58] [Q&A] [answered]
How to perform transactions in case of postgresql? It would be nice to add an example in repo and docs.

**Comment by matt-42** [comment, marked as answer]
Hi @shivshankardayal ,

Here is one example of transactions:
```c++
connection("START TRANSACTION;");
// [... your requests... ]
connection("COMMIT;");
```

Thanks for the feedback, I'll add an example in the doc.

Don't hesitate if you have other questions
